### PR TITLE
Fix buttons alignments on modal on import page

### PR DESCRIPTION
**Pull request by antoin-m**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The buttons in the modal of the import page should be aligned in the bottom right
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-513
| How to test?  | Import something.

### CODE REVIEW

File: admin-dev/themes/default/template/controllers/import/modal_import_progress.tpl [hunk 1]
```diff
@@ -82,17 +82,19 @@
...
-		</button>
-		&nbsp;
-		<button type="button" class="btn btn-success" data-dismiss="modal" tabindex="-1" id="import_close_button" style="display: none;">
-			Close
-		</button>
-	</div>
+  <div class="modal-footer">
+    <div class="input-group pull-right">
+      <button type="button" class="btn btn-primary" tabindex="-1" id="import_continue_button" style="display: none;">
+        Ignore warnings and continue?
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/template/controllers/import/modal_import_progress.tpl]: no trads ? /c @AlexEven

**Comment by aleeks** [comment]
Need trans here ?

**Comment by aleeks** [comment]
Hello @antoin-m 
Can you add some trans according with @AlexEven please ?!!

We need to merge this.

File: admin-dev/themes/default/template/controllers/import/modal_import_progress.tpl [hunk 2]
```diff
@@ -82,17 +82,19 @@
...
+      </button>
+      &nbsp;
+      <button type="button" class="btn btn-default" tabindex="-1" id="import_stop_button">
+        Abort import
+      </button>
+      &nbsp;
+      <button type="button" class="btn btn-success" data-dismiss="modal" tabindex="-1" id="import_close_button" style="display: none;">
+        Close
```
Comment (AlexEven) [inline admin-dev/themes/default/template/controllers/import/modal_import_progress.tpl]: translation domain: Admin.Actions

File: admin-dev/themes/default/template/controllers/import/modal_import_progress.tpl [hunk 3]
```diff
@@ -82,17 +82,19 @@
...
-		</button>
-	</div>
+  <div class="modal-footer">
+    <div class="input-group pull-right">
+      <button type="button" class="btn btn-primary" tabindex="-1" id="import_continue_button" style="display: none;">
+        Ignore warnings and continue?
+      </button>
+      &nbsp;
+      <button type="button" class="btn btn-default" tabindex="-1" id="import_stop_button">
+        Abort import
```
Comment (AlexEven) [inline admin-dev/themes/default/template/controllers/import/modal_import_progress.tpl]: translation domain: Admin.AdvParameters.Feature

File: admin-dev/themes/default/template/controllers/import/modal_import_progress.tpl (hunk 1 above)
Comment (AlexEven) [inline admin-dev/themes/default/template/controllers/import/modal_import_progress.tpl]: Yes, all the strings should be translatable, thanks @mickaelandrieu for spotting this.
translation domain: Admin.AdvParameters.Notification

Thank you!

**Comment by aleeks** [comment]
Thank you bro @antoin-m

**Pull request merged**